testsuite/demo/run-pass/channel.verona

```diff
@@ -0,0 +1,156 @@
+/*****************************************************************************
+ * This example demonstrates how one could build a channel.
+ *
+ * It illustrates creating and using promises and cowns and communicating
+ * between behaviours.
```

Comment:
Does it?

Reply:
As well as cowns, the implementation uses promises to communicate between behaviours.